core/mastodon/src/test/java/br/com/orcinus/orca/core/mastodon/instance/requester/RequesterTestScope.kt

```diff
@@ -80,12 +83,14 @@ internal inline fun retryCountOf(
  *
  * @param clientResponseProvider Defines how the [HttpClient] will respond to requests.
  * @param onAuthentication Action run whenever the [Actor] is authenticated.
+ * @param context [CoroutineContext] in which [Job]s are launched by default.
  * @param body Operation to be performed with the [Requester].
  */
 @OptIn(ExperimentalContracts::class)
 internal inline fun runRequesterTest(
   clientResponseProvider: ClientResponseProvider = ClientResponseProvider.ok,
   crossinline onAuthentication: () -> Unit = {},
+  context: CoroutineContext = EmptyCoroutineContext,
```

Comment:
Ideally, as this change is unrelated, it should not be in this PR. Nevertheless, there should be a test asserting that the body indeed does run in the specified context.